libs/juno-ui-components/src/components/Checkbox/Checkbox.component.js

```diff
@@ -1,51 +1,144 @@
-import React from "react"
+import React, { useState, useEffect } from "react"
 import PropTypes from "prop-types"
 
-const checkboxstyles = `
-	rounded-full
+// TODO: disabled styles missing
+
+const inputstyles = `
+	w-4
+	h-4
+	opacity-0
+	cursor-pointer
```

Comment:
Shall we add `z-50` to ensure the input has the highest z-index?

Reply:
I usually try to only use z-index when it is really needed, but if you want no prob?